### PR TITLE
Refactoring: remove abbreviations, run formatter

### PR DESCRIPTION
**Pull request by mrks**
This just replaces a couple of abbreviations and fixes a number of code style violations. I don't think that a thorough review is needed.

### CODE REVIEW

File: src/lib/utils/cuckoo_hashtable.hpp [hunk 1]
```diff
@@ -57,9 +57,9 @@ class HashTable : private Noncopyable {
   std::shared_ptr<PosList> get(S value) {
     for (size_t i = 0; i < NUMBER_OF_HASH_FUNCTIONS; i++) {
       auto position = hash<S>(i, value);
-      auto elem = _hashtables[i][position];
-      if (elem != nullptr && value_equal(elem->value, value)) {
-        return elem->row_ids;
+      auto element = _hashtables[i][position];
+      if (element != nullptr && value_equal(element->value, value)) {
+        return element->row_ids;
```
Comment (mjendruk) [inline src/lib/utils/cuckoo_hashtable.hpp]: I think most of the changes make sense. IMHO, `elem` is clear enough, though. At another place, you changed `iter` to `it`, which is even shorter.

File: src/lib/storage/index/group_key/variable_length_key_base.cpp [hunk 2]
```diff
@@ -69,32 +69,34 @@ VariableLengthKeyBase& VariableLengthKeyBase::shift_and_set(uint64_t value, uint
   return *this;
 }
 
-bool operator==(const VariableLengthKeyBase& lhs, const VariableLengthKeyBase& rhs) {
```
Comment (mjendruk) [inline src/lib/storage/index/group_key/variable_length_key_base.cpp]: `lhs` and `rhs` are fairly common abbreviatinos. Why change it?

Comment (mrks) [reply src/lib/storage/index/group_key/variable_length_key_base.cpp]: I'm fine with lhs / rhs as well, but it should be consistent everywhere. I just realized that I missed some places. Unless you're a big fan of lhs, I would change the rest as well.

File: src/lib/utils/cuckoo_hashtable.hpp (hunk 1 above)
Comment (mrks) [reply src/lib/utils/cuckoo_hashtable.hpp]: `_it` is used in most places. I am fine with either, but this should also be consistent.

Comment (mjendruk) [reply src/lib/utils/cuckoo_hashtable.hpp]: Okay should this and `left` / `right` be added to the glossary?

Comment (mrks) [reply src/lib/utils/cuckoo_hashtable.hpp]: Added `_it`. I don't think that left/right fits.

**Pull request merged**